lib/emque/producing/message/message.rb

```diff
@@ -36,6 +36,16 @@ def read_raise_on_failure
           end
         end
 
+        def ignored_exceptions(*ignored_exceptions)
+          @ignored_exceptions = ignored_exceptions
+        end
+
+        def read_ignored_exceptions
+          (Array(@ignored_exceptions) +
+            [InvalidMessageError, MessagesNotSentError] +
```

Comment:
Yes indeed... good spot. Let's go for that, but I suggest putting `MessageNotSentError` into the array in Emque::Producing::Configuration#initialize so that it can be removed during app configuration.